java/build/bazel/tests/integration/WorkspaceDriver.java

```diff
@@ -155,6 +151,27 @@ protected void copyFromRunfiles(String path, String destpath) throws IOException
     Files.copy(origin.toPath(), dest.toPath());
   }
 
+  /**
+   * Copy the whole directory from the runfiles under {@code rootDirectoryPath} to the current workspace.
+   */
+  protected void copyFromRunfilesDirectory(final String rootDirectoryPath) throws IOException {
+    File root = getRunfile(rootDirectoryPath);
+    try (Stream<Path> paths = Files.walk(Paths.get(root.toURI()))) {
+      paths.filter(path -> new File(path.toUri()).isFile())
```

Comment:
the java 8 way to do `new File(path.toUri()).isFile()` is with `java.nio.file.Files.isRegularFile`